gist/src/main/java/build/gist/data/listeners/Queue.kt

```diff
@@ -68,6 +68,8 @@ class Queue: GistListener {
             try {
                 Log.i(GIST_TAG, "Fetching user messages")
                 val latestMessagesResponse = gistQueueService.fetchMessagesForUser()
+                // To prevent us from showing expired / revoked messages, clear user messages from local queue.
+                clearUserMessagesFromLocalStore()
```

Comment:
Can this function call go next to the caller of `fetchUserMessages`? From reading the function `fetchUserMessages`, it makes me think the function should only be fetching messages and not doing anything to the storage of messages. 

Especially since I don't see messages being stored (just cleared) in the `fetchUserMessages` function body, I am afraid that this function call here could create some edge case bugs later on.

Reply:
fetchUserMessages gets messages from the Gist message queue service and either displays them (if all conditions are met) or stores them locally.

In this case, we only want to clear the local store if the call succeeds.

Reply:
I agree, we'll address this in a future PR. Thanks for the review.